src/viewPanel/gridStackItem.ts

```diff
@@ -63,7 +62,11 @@ export class GridStackItem extends Panel {
 
   private _createToolbar(itemTitle: string): Toolbar {
     const toolbar = new Toolbar();
+    toolbar.node.addEventListener('click', () => {
```

Comment:
It would feel nicer to add the event listener to the main container of the gridstack item